libs/components/layout/src/lib/modules/box/box.component.scss

```diff
@@ -17,7 +18,11 @@ sky-box {
 }
 
 .sky-box {
-  background-color: #fff;
```

Comment:
background color is already set by the global box class, so this wasn't necessary. I'm only setting the custom property here because the global class is used in places where we won't want to change it